src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -232,6 +233,63 @@ namespace AirflowNetworkBalanceManager {
 	Real64
 	GetZoneInfilAirChangeRate( int const ZoneNum ); // hybrid ventilation system controlled zone number
 
+	// derived class or struct
+	struct OccupantVentilationControlProp {
+
```

Comment:
@lgu1234 Declaring OpeningStatus as an OpenStatus enum rather than as an int is what you need to do:

    OpenStatus OpeningStatus;

So the name is really not great, kind of confusing. Can you think of a more descriptive name for the enum? I tried and failed.

We definitely don't want to be doing the int() conversion.